### ISSUE TITLE
The word R1A_CVHSA isn't defined, so it can't name a protein.

### ISSUE THREAD

**Issue by aymanosman** [ddmcdonald/sparser#3]
I get the following error when trying to load the system.

```
While evaluating the form starting at line 169, column 0
  of #P"/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/protein-fam-no-id.lisp":
While evaluating the form starting at line 118, column 0
  of #P"/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/loader.lisp":

debugger invoked on a SIMPLE-ERROR in thread
#<THREAD "main thread" RUNNING {706C863083}>:
  The word R1A_CVHSA isn't defined, so it can't name a protein.

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY                        ] Retry EVAL of current toplevel form.
  1: [CONTINUE                     ] Ignore error and continue loading file "/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/protein-fam-no-id.lisp".
  2: [ABORT                        ] Abort loading file "/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/protein-fam-no-id.lisp".
  3:                                 Retry EVAL of current toplevel form.
  4:                                 Ignore error and continue loading file "/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/loader.lisp".
  5:                                 Abort loading file "/Users/ayman/git/github.com/ddmcdonald/sparser/Sparser/code/s/grammar/model/sl/biology/loader.lisp".
  6: [TRY-RECOMPILING              ] Recompile Sparser/code/s/init/everything and try loading it again
  7: [RETRY                        ] Retry
                                     loading FASL for #<CL-SOURCE-FILE "sparser" "Sparser/code/s/init/everything">.
  8: [ACCEPT                       ] Continue, treating
                                     loading FASL for #<CL-SOURCE-FILE "sparser" "Sparser/code/s/init/everything">
                                     as having been successful.
  9:                                 Retry ASDF operation.
 10: [CLEAR-CONFIGURATION-AND-RETRY] Retry ASDF operation after resetting the
                                     configuration.
 11:                                 Retry ASDF operation.
 12:                                 Retry ASDF operation after resetting the
                                     configuration.
 13:                                 Give up on "sparser"
 14: [REGISTER-LOCAL-PROJECTS      ] Register local projects and try again.
 15:                                 Exit debugger, returning to top level.

((:METHOD GET-PROTEIN (STRING)) "R1A_CVHSA") [fast-method]
   source: (ERROR "The word ~a isn't defined, so it can't ~
                name a protein."
                  PNAME)
```

**Comment by ddmcdonald** [comment]
This problem is fixed in commit 3d0963f, made by Robert Bobrow on November  10th"Revised the code that threw an error ...".  to the file mechanics.lisp in grammar/model/sl/biology/.
The underlying problem is that all the local developers have a parameter set in their init file that controls whether all the proteins are loaded. There was no handler for the case where this parameter did not have a value. The just-pushed modifications to the toplevel ReadMe should make this clearer

**Comment by aymanosman** [comment]
Thanks, loaded successfully.
